src/receiver/ltp.ts

```diff
@@ -0,0 +1,26 @@
+import { DMXUniverse } from '../data/universe';
+import { PreparedData } from '../data/merging';
+import { AbstractMergingReceiver } from './abstract';
+
+export class LTPMergingReceiver extends AbstractMergingReceiver {
+  // eslint-disable-next-line class-methods-use-this
+  protected mergeData(data: PreparedData): DMXUniverse | null {
+    const { referenceData } = data.universeData;
+    const mergedData = new DMXUniverse();
+
+    for (let ch = 0; ch < 512; ch += 1) {
+      let referenceTime = 0;
+      mergedData.data[ch] = referenceData.data[ch] || 0;
+
+      for (const [, tmp] of data.universeData.servers) {
+        if (tmp.lastUpdate > referenceTime) {
```

Comment:
From https://github.com/k-yle/sACN/pull/58#issuecomment-1988327213:
> #### LTP
> [...]
> - In case of _different priorities_:
>    The sender using the highest priority wins for the whole universe.

That sounds logical to me. However, the LTP algorithm doesn't consider the `priority` right now - I think we need something like this: 

```suggestion
        if (tmp.lastUpdate > referenceTime && tmp.priority === data.maximumPriority) {
```

---

Also, it seems like the LTP algorithm has no way to reset a channel value to `0`? Once a channel is set to a non-0 value, it is impossible to set that channel back to `0`. I'm interested to know when this behaviour is useful?